client/extensions/belgaCoverage/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import {ISuperdesk, IExtension, IExtensionActivationResult} from 'superdesk-api';
```

Comment:
If you committed a rename without any changes and added changes in another commit I think git would be able to 
 preserve the diff.